reference-architectures/deploy-arch-ibm-slz-vpn.md

```diff
@@ -0,0 +1,100 @@
+---
+
+copyright:
+  years: 2025
+lastupdated: "2025-03-05"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: "Andrej Kocbek"
+
+# The release that the reference architecture describes
+version: 2.1.3
+
+# Whether the reference architecture is published to Cloud Docs production.
+# When set to false, the file is available only in staging. Default is false.
+production: false
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page
+# for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-client-to-site-vpn-1b824983-263f-4191-bfcd-c1d1b2220aa3-global
+
+docs: https://cloud.ibm.com/docs/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-connect-landingzone-client-vpn
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/main/reference-architectures/c2s-basic.drawio.svg
```

Comment:
diagram is inside VPN repo under `reference-architectures`. Will automation will be able to pick it up? https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/main/reference-architectures/c2s-basic.drawio.svg